ADAL/src/public/ADTelemetry.h

```diff
@@ -76,15 +57,15 @@
 /*!
```

Comment:
whoops, wrong line...I meant here:

https://github.com/AzureAD/azure-activedirectory-library-for-objc/pull/859/files#diff-53fd7a8529454913eb5701503ea1a067R49